_pages/testing.md

```diff
@@ -13,16 +13,37 @@ Login.gov provides an open sandbox environment to create and test integrations b
+Login.gov does not manage user accounts in the sandbox.
 
-1. Visit the dashboard at [https://dashboard.int.identitysandbox.gov](https://dashboard.int.identitysandbox.gov). Select the “Sign in” button in the upper-right corner to sign in or create a new account with the test Login.gov IdP in the sandbox environment hosted at [idp.int.identitysandbox.gov](https://idp.int.identitysandbox.gov). Please note that this is separate from your Login.gov production account. 
-2. Once you are logged into your sandbox account, create a new team by selecting the “Continue” button under “Create your first team." (If you have previously created a team you will select the “Create a new team” button.) Next you can add users to that team.
-3. After creating your team, select the Apps tab. This page is where you will find all of the test applications you and your team create.
-4. Select the “Create a new test app” button and fill out the form to register a new application with the Login.gov IdP in the test sandbox environment.
-5. Start testing! If you need to troubleshoot an issue that is not covered in the [developer documentation]({% link _pages/index.md %}), please submit a support ticket through the [Partner Support Help Desk](https://zendesk.login.gov). We can also add you to our partner support Slack channel and the Login.gov team will help you along the way.
-6. When you're ready to go to production, please [follow our production deployment instructions]({% link _pages/production.md %}). We'll manage your application's promotion to production. The move to production may take up to two weeks.
+**If you are an agency partner with a .gov or .mil account:**
+You can create an account in the sandbox environment on your own.
+
+**If you are a government contractor:**
+Ask your agency partner to help you gain access. Login.gov will not create an account or add you to a team; your partner must do this for you.
+
+**If you are with a government entity that is not a federal agency (a state or municipality) and do not have an email ending in .gov or .mil**
+Please submit a support ticket through the [Partner Support Help Desk](https://zendesk.login.gov) to get access to the Dashboard.
+
+## Using the sandbox
+
+1. Visit the Partner Dashboard at [https://dashboard.int.identitysandbox.gov](https://dashboard.int.identitysandbox.gov).
+1. Select the “Sign in” button in the upper-right corner to sign in or create a new account with the test Login.gov IdP in the sandbox environment hosted at [idp.int.identitysandbox.gov](https://idp.int.identitysandbox.gov). **Please note that this is separate from your Login.gov production account.**
```

Comment:
Why are these all `1.` Is that something that will automatically number itself then it gets rendered?

Reply:
yes! markdown numbers automatically, so it's a bit easier to edit if it's all `1` but a bit confusing to review. sorry about that. you can see it numbered correctly in the build here: https://federalist-14ccf384-8bd5-4e30-aa95-a5fe5894e3c1.sites.pages.cloud.gov/preview/18f/identity-dev-docs/dmm/clarify-dashboard-access/testing/#using-the-sandbox

Reply:
Thats what I figured, but wanted a sanity check.